packages/php/blueprint/src/Importers/ImportInstallTheme.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Automattic\WooCommerce\Blueprint\Importers;
+
+use Automattic\WooCommerce\Blueprint\ResourceStorages;
+use Automattic\WooCommerce\Blueprint\StepProcessor;
+use Automattic\WooCommerce\Blueprint\StepProcessorResult;
+use Automattic\WooCommerce\Blueprint\Steps\InstallTheme;
+use Automattic\WooCommerce\Blueprint\UseWPFunctions;
+use Plugin_Upgrader;
+
+class ImportInstallTheme implements StepProcessor {
+	use UseWPFunctions;
+	private ResourceStorages $storage;
+	private StepProcessorResult $result;
+
+	public function __construct( ResourceStorages $storage ) {
+		$this->result  = StepProcessorResult::success( 'InstallThemes' );
```

Comment:
Nit: should we call it `InstallTheme` instead?

Reply:
Updated in [Use the correct step names](https://github.com/woocommerce/woocommerce/pull/49763/commits/998c76341c3cb81d94781d5e5e0cb047aa30a86b)